roles/kubeadm_configs_compute/tasks/main.yaml

```diff
@@ -55,12 +55,41 @@
   changed_when: false
   check_mode: false
   delegate_to: '{{ groups.cp_running|first }}'
-  register: in_cluster_kubelet_config
+  register: in_cluster_kubelet_config_versionned
   when:
     - found_kubectl.rc == 0
     - groups.cp_running|default([])|length > 0
+    - current_cp_version is version("1.25", "<")
   environment:
     KUBECONFIG: /etc/kubernetes/admin.conf
+  ignore_errors: '{{ current_cp_version is version("1.23", ">=") }}'
```

Comment:
May be using the kubectl option:
```
--ignore-not-found=false:
If the requested object does not exist the command will return exit code 0.
```
could be used to prevent this `ignore_errors`